### DISCUSSION TITLE
Bing Co-pilot in Pieces

### DISCUSSION THREAD

**Discussion by Perspective2077** [pieces-app/opensource#43] [Contribution Ideas]
I love Bing co-pilot because it the only co-pilot imo that doesn't get manipulated if we try too , and also and give the correct answer  , So It would be awesome if u add bing ai in pieces :)

**Comment by Sophyia7** [comment]
@I-am-Perspective  Thanks for this idea. Bing AI is trained on GPT-4, Prometheus, and DALL-E 3, and we have GPT-4. However, you can request other LLMs that Bing AI uses here: https://github.com/pieces-app/support/discussions/121
